includes/assets.php

```diff
@@ -243,6 +243,30 @@ public function register_scripts() {
 				);
 			}
 
+			// Register core scripts
```

Comment:
There is most likely some code duplication as the same script handles are listed above. @cbravobernal should have more insights here. I also think that we could reuse the asset files generate in the build process together with `commands-admin` and `commands-custom-post-types`. I also belive this script handles need to be prefixed as it is only possible to register one script under a given script handle. These names are too general.

Reply:
Thanks for catching that, they are indeed a leftover after tests.

I've also prefixed the assets again with `scf-` to avoid potential collisions with other plugins.